tests/robot-tests/tests/libs/common.robot

```diff
@@ -773,13 +773,9 @@ user closes Set Page View box
     user waits until element is not visible    id:editingMode
 
 user checks page for details section
```

Comment:
This keyword can now be renamed to `user waits until page contains details section`

Although if nothing is using this keyword then I think it can be removed?